src/write_csv.jl

```diff
@@ -15,12 +15,18 @@ function write_csv(df::DataFrame, erupt_saved::EruptSaved{Float64}, path::String
     ]
     rename!(df, ["timestamp" => "time"])
     rename!(df, ["value$i" => names[i] for i in 1:10])
+    # add eps_x column
+    m_h2o = df[:, "total_mass_H2O"] ./ df[:, "total_mass"]
+    m_co2 = df[:, "total_mass_CO2"] ./ df[:, "total_mass"]
+    crystal_fraction_eps_x′(T, P, mH2O, mCO2) = crystal_fraction_eps_x(composition, T, P, mH2O, mCO2)
+    df[!, "eps_x"] = crystal_fraction_eps_x′.(df[:, "T"], df[:, "P+dP"], m_h2o, m_co2)
```

Comment:
Is `crystal_fraction_eps_x′` used more than once? If not, then it would be simpler to use the original `crystal_fraction_eps_x`.

Reply:
This function is used only one time. I'll use the original function for it. Thanks for your comment!